pkgs/shells/zsh/zsh-powerlevel10k/default.nix

```
@@ -5,12 +5,12 @@
 
 stdenv.mkDerivation {
   pname = "powerlevel10k";
-  version = "1.1";
+  version = "1.2.0";
   src = fetchFromGitHub {
     owner = "romkatv";
     repo = "powerlevel10k";
-    rev = "9d9c50611da19044370ee759e593ccadbad32a6a";
-    sha256 = "0jl4jamh7i8w7lp3qbbakh1wsyrrmslsrjwh9jgyvw50a83hp6fj";
+    rev = "cb3f58d21f643d26d71f520d61a592816cca3edf";
```

Would it make sense to just substitute in the `version`? They both point to the same commit, but this way the `rev` shouldn't need to be updated every time.

```suggestion
    rev = "v${version}";
```

---

I assume this was mostly a leftover from the time when powerlevel10k didn't do releases - we should indeed do as suggested here.

---

Indeed, i pushed an update.